File: javascript/1.js
let arr=[2,3,5,1,4];
let evenSum=0;
let oddSum=0;
arr.forEach((ele,ind)=>{
    if((ele%2)===0 && (ind%2===0)){
        evenSum+=ele;
    }
    else if((ele%2)!=0 && (ind%2!=0)){
        oddSum+=ele
    }
})
console.log(evenSum);
console.log(oddSum);

File: python/1.py
n = int(input())
arr = list(int(i) for i in input().strip().split(' '))
ind=0
evensum=0
oddsum=0
for a in arr:
    if a%2==0 and ind%2==0:
        evensum+=a
    elif a%2==1 and ind%2==1:
        oddsum+=a
    ind+=1
print(evensum,' ',oddsum)
